nntrainer/dataset/batch_queue.cpp

```diff
@@ -68,4 +68,80 @@ bool BatchQueue::isEmpty() const {
   return q.empty();
 }
 
+IterationQueue::IterationQueue(
+  unsigned int num_slots, const std::vector<ml::train::TensorDim> &input_dims,
+  const std::vector<ml::train::TensorDim> &label_dims) {
```

Comment:
initialize being_filled.

Reply:
This is initialized in the later PR. I'll skip updating it here to avoid unnecessary conflict.

Reply:
https://github.com/nnstreamer/nntrainer/pull/1488/commits/f089ae13ca53c04d4c8e882dd5a197a8e75b03ce#diff-45e2e03d99d3d4d24a28897fb30cf19bf900f94742302f6b8420b0d1717dbaf0R73-R77